src/app/components/import-annotations/details/details.component.html

```diff
@@ -0,0 +1,243 @@
+<h1>Import Annotations: {{ audioEventImport.name }}</h1>
+
+<p
+  *ngIf="audioEventImport.descriptionHtml; else defaultDescription"
+  [innerHTML]="audioEventImport.descriptionHtml"
+></p>
+
+<div>
+  <section>
+    <h2>Events</h2>
+    <ngx-datatable
+      bawDatatableDefaults
+      [bawDatatablePagination]="{ filters: filters$, getModels: getModels }"
+    >
+      <ngx-datatable-column prop="">
+        <ng-template let-column="column" ngx-datatable-header-template>
+          Audio Recording
+        </ng-template>
+
+        <ng-template let-value="value" ngx-datatable-cell-template>
+          <baw-loading
+            *ngIf="
+              value.audioRecording | isUnresolved;
+              else audioRecordingTemplate
+            "
+            size="sm"
+          ></baw-loading>
+
+          <ng-template #audioRecordingTemplate>
+            <a [bawUrl]="value.audioRecording.viewUrl">
+              {{ value.audioRecording.id }}
+            </a>
+          </ng-template>
```

Comment:
it occurs to me that we could have loading component that is so much smarter than our current one:

```suggestion
          <baw-loading value="value.audioRecording">
                <a [bawUrl]="value.audioRecording.viewUrl">
                      {{ value.audioRecording.id }}
                </a>
          </baw-loading>
```

Reply:
I like this syntax.

Should we add this to the scope of this PR or do it in another "code cleanup" PR with the `successMsg` and `errorMsg` form template refactoring (change name from `successMsg` -> `successMessage` & `errorMsg` -> `errorMessage`)?

Reply:
Separate PR